app/views/_cascade/formats/modular/ad_landing/widgets/_masthead_hero.html.erb

```diff
@@ -0,0 +1,99 @@
+<%#
```

Comment:
Would be nice if we could just copy over the Cascade format directly. But we'd need a Velocity parser or transpiler. Volunteers?